src/app-layer-ftp.c

```diff
@@ -190,12 +190,14 @@ static int FTPParseRequestCommand(void *ftp_state, uint8_t *input,
         if (SCMemcmpLowercase("port", input, 4) == 0) {
             fstate->command = FTP_COMMAND_PORT;
         }
+    }
 
-        /* else {
-         *     Add the ftp commands you need here
-         * }
-         */
+    if (input_len >= 8) {
```

Comment:
minor: this can be a single line: if (input_len >= 8 && SCMemcmpLowercase("auth tls", input, 8) == 0) {